test/integration/github.test.js

```diff
@@ -135,16 +136,19 @@ describe('github integration tests', () => {
     });
   });
   describe('pull requests from user with a signed Adobe ICLA (account adobeiotest1)', () => {
-    const user = 'adobeiotest1';
+    const user = 'adobetester1';
     const newBranch = '' + new Date().valueOf();
     const github = new Octokit({
       auth: process.env.TEST_ONE_PAC
     });
+    // use only on this test first
     it('should approve a pull request to an adobe repo', async () => {
+      const userdata = await github.rest.rateLimit.get();
+      console.log(userdata.data.resources.core);
```

Comment:
Remove this console.log?

Reply:
useful for debugging the integration tests, it doesn't run right now, so it doesn't matter